org-roam-protocol.el

```diff
@@ -39,19 +39,12 @@
 (require 'org-protocol)
 (require 'org-roam)
 
+;;;; Declarations
+(defvar org-roam-capture-ref-templates)
```

Comment:
The default template seems to be gone here.

Reply:
That variable was moved to org-roam-capture.el: 
https://github.com/progfolio/org-roam/blob/70bee2679080e6752207988a2084f0671c4bfc0c/org-roam-capture.el#L106-L114

We can actually remove that entire set of declarations since we're requiring org-roam which requires org-roam-capture itself.